src/http/http-driver.ts

```diff
@@ -8,13 +8,22 @@ export class HttpDriver implements Driver {
     private authHeader: string | undefined;
 
     constructor(url: URL) {
+        for (const [key, value] of url.searchParams.entries()) {
```

Comment:
I was thinking the JWT would come from the config object, which is a *lot* easier for the developer than encoding it into the URL.